js/package.json

```diff
@@ -13,6 +13,7 @@
   "scripts": {
     "bootstrap": "lerna link && lerna bootstrap",
     "build": "lerna run build",
+    "build-CI-without-gumdrop": "lerna run build --ignore gumdrop",
```

Comment:
gumdrop causes Vercel to run out of memory. leaving out of CI until we move it all out of Vercel